### PR TITLE
Switch from GitLab CI to Github Actions

### PR DESCRIPTION
**Pull request by jhkennedy**

### CODE REVIEW

**Comment by jhkennedy** [comment]
@asjohnston-asf and @Jlrine2 Pulling this into your branch would also get your HyP3v2 work going on GitHub

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+      - name: Export PR body
+        id: pr
+        run: |
+          PR_QUERY='.data.repository.commit.associatedPullRequests.edges[0].node.body'
+          PR_BODY=$(jq --raw-output "${PR_QUERY}"  pr.json)
+          echo "::set-output name=body::${PR_BODY}"
+
+      - name: Create Release
+        id: create_release
```
Comment (asjohnston-asf) [inline .github/workflows/release.yml:37]: I don't see this id being referenced anywhere, could drop it

Comment (jhkennedy) [reply .github/workflows/release.yml:37]: It's being used here: https://github.com/asfadmin/hyp3-rtc-gamma/pull/85/files/817c00f4062a3bee136e6b4391b51328dd2ceb26#diff-56c30d9224897b08c28075052d4595c5R44

Comment (asjohnston-asf) [reply .github/workflows/release.yml:37]: that's the id from the preceding `Export PR body` step, not the `Create Release` step?

Comment (jhkennedy) [reply .github/workflows/release.yml:37]: Your right -- it's in their suggested usage but it's not needed.

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      - v*
+
+
+jobs:
+  finish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: '0'
```
Comment (asjohnston-asf) [inline .github/workflows/release.yml:15]: drop quotes

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  push:
+    tags:
+      - v*
+
+
+jobs:
+  finish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```
Comment (asjohnston-asf) [inline .github/workflows/release.yml:13]: preceding blank line?

File: .github/workflows/release.yml [hunk 4]
```diff
@@ -0,0 +1,65 @@
+name: Write release and finish
+
+on:
+  push:
+    tags:
+      - v*
+
+
+jobs:
+  finish:
```
Comment (asjohnston-asf) [inline .github/workflows/release.yml:10]: preceding blank line?

File: .github/workflows/static-analysis.yml [hunk 5]
```diff
@@ -0,0 +1,58 @@
+name: Static analysis
+
+on: push
+
+jobs:
+  flake8:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
```
Comment (asjohnston-asf) [inline .github/workflows/static-analysis.yml:9]: indentation

File: .github/workflows/static-analysis.yml [hunk 6]
```diff
@@ -0,0 +1,58 @@
+name: Static analysis
+
+on: push
+
+jobs:
+  flake8:
```
Comment (asjohnston-asf) [inline .github/workflows/static-analysis.yml:6]: preceding blank line?

File: .github/workflows/static-analysis.yml (hunk 5 above)
Comment (asjohnston-asf) [inline .github/workflows/static-analysis.yml:9]: preceding blank line?

File: .github/workflows/static-analysis.yml [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+        flake8 --max-line-length=120 --import-order-style=pycharm --statistics \
+            --application-import-names hyp3_rtc_gamma
+
+
+  gitleaks:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: '0'
```
Comment (asjohnston-asf) [inline .github/workflows/static-analysis.yml:31]: drop quotes

File: .github/workflows/static-analysis.yml [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+    - name: Lint with flake8
+      run: |
+        flake8 --max-line-length=120 --import-order-style=pycharm --statistics \
+            --application-import-names hyp3_rtc_gamma
+
+
+  gitleaks:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```
Comment (asjohnston-asf) [inline .github/workflows/static-analysis.yml:29]: preceding blank line?

File: .github/workflows/static-analysis.yml [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+      - name: Scan for secrets with gitleaks
+        uses: zricethezav/gitleaks-action@master
+
+
+  trufflehog:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: '0'
```
Comment (asjohnston-asf) [inline .github/workflows/static-analysis.yml:42]: drop quotes

File: .github/workflows/test-and-build.yml [hunk 10]
```diff
@@ -0,0 +1,189 @@
+name: Test and build
+
+on:
+  push:
+    branches:
+    - master
```
Comment (asjohnston-asf) [inline .github/workflows/test-and-build.yml:6]: indentation

File: .github/workflows/test-and-build.yml [hunk 11]
```diff
@@ -0,0 +1,189 @@
...
+
+  package:
+    runs-on: ubuntu-latest
+    needs: pytest
+    outputs:
+      SDIST_VERSION:  ${{ steps.build.outputs.version }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: '0'
```
Comment (asjohnston-asf) [inline .github/workflows/test-and-build.yml:55]: drop quotes

**Comment by jhkennedy** [comment]
@asjohnston-asf Thanks!

**Pull request merged**